packages/modeling/src/operations/booleans/trees/splitPolygonByPlane.js

```diff
@@ -36,7 +36,7 @@ const splitPolygonByPlane = (splane, polygon) => {
     const MINEPS = -EPS
     for (let i = 0; i < numvertices; i++) {
       const t = vec3.dot(splane, vertices[i]) - splane[3]
-      const isback = (t < 0)
+      const isback = (t < MINEPS)
```

Comment:
I had to think for a minute about why this change. It guess it needs to match line 42 though.